content/messaging/customer-chat-api/changelog/index.mdx

```diff
@@ -48,6 +48,12 @@ This document is the record of all the changes in the **Customer Chat API**, Web
   - the `group_id` parameter was renamed to `id`.
 - In the [**List License Properties**](/messaging/customer-chat-api/v3.3/#list-license-properties) method, the `license_id` parameter was removed.
 
+### Other
+
+- The **Get Predicted Agent** ([Web](/messaging/customer-chat-api/v3.3/#get_predicted_agent) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#get_predicted_agent)) response has a new format:
+  - previous fields are now nested together as the `agent` field,
```

Comment:
```suggestion
  - The fields are now nested together as the `agent` field.
```

Reply:
"Previous fields" suggests that there are new fields that replaced the old ones, while, in fact, they still exist, but have a different form.